.devcontainer/devcontainer.json

```diff
@@ -33,5 +40,6 @@
 		"PULSE_SERVER": "/mnt/wslg/PulseServer",
 		"WAYLAND_DISPLAY": "wayland-0",
 		"XDG_RUNTIME_DIR": "/mnt/wslg/runtime-dir"
-	}
+	},
+	"shutdownAction": "stopCompose",
```

Comment:
Can this shutdown action be a command? Or is there an option that runs `docker compose down` instead of `stop`?

Reply:
You can see the log we get in the linked issue; the VSCode plugin seems to try to `docker-compose stop` and fail. Copy/pasting the same command in WSL does stop the container. I haven't been able to see the log from the failed command by the plugin.